provision/rosa-cross-dc/Taskfile.yaml

```diff
@@ -17,6 +17,7 @@ vars:
   BENCHMARK_DIR: "{{.ROOT_DIR}}/../../benchmark/src/main/content/bin"
   KEYCLOAK_MASTER_PASSWORD:
     sh: aws secretsmanager get-secret-value --region eu-central-1 --secret-id keycloak-master-password --query SecretString --output text --no-cli-pager
+  KC_JGROUPS_TLS: '{{default "true" .KC_JGROUPS_TLS}}'
```

Comment:
The global variable is not required. If you want to keep it, the line 280 can be simplified:
`--set infinispan.jgroupsTls={{ .KC_JGROUPS_TLS }}`

Reply:
Thanks. I would like to keep it as it serves as some kind of reference at the top of the file. And I'd like to keep it `true`. 

Simplified the setup below as you suggested.